profiles/common/themes/europa/styleguide/assets/public/prettify.js

```diff
@@ -1,5 +1,8 @@
-// Copyright (C) 2006 Google Inc.
-//
+/**
+ * @file
+ * Copyright (C) 2006 Google Inc.
+ */
+
 // Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This entire file is third-party code, there is no need to fix the coding standards in this. We should just exclude it by adding it to the list of third-party code.